lib/commands/serve.js

```diff
@@ -21,9 +18,9 @@ module.exports = Command.extend({
     {
       name: 'port',
       type: Number,
-      default: defaultPort,
+      default: process.env.PORT || DEFAULT_PORT,
```

Comment:
I got rid of `DEFAULT_PORT` from the default in the options originally to detect if it was left blank, perhaps this would be better? @NullVoxPopuli 
```suggestion
      default: process.env.PORT ?? 0,
```

Reply:
the issue I see is that it doesn't communicate that the actual default is 4200.

when you say `0`, I think you're talking more about the _behavior_ when 4200 is taken -- which is more of an implementation detail of the behavior imo

Reply:
I'll keep 4200 on the default param and check for `DEFAULT_PORT` on line 108 instead